src/native/custom_key_op_handler.c

```diff
@@ -4,23 +4,51 @@
  */
 #include "custom_key_op_handler.h"
 
+#include "java_class_ids.h"
+#include "tls_context_pkcs11_options.h"
+#include <aws/common/string.h>
+#include <aws/io/tls_channel_handler.h>
+
+struct aws_jni_custom_key_op_handler {
+    JavaVM *jvm;
+
+    /**
+     * A reference to the Java class that this struct is linked to.
+     * The interface, strings, etc, can be gotten from this class.
+     */
+    jobject jni_custom_key_op;
+
+    /**
+     * The C class containing the key operations. We extend/define it's VTable to allow
+     * us to have it call into the customer's Java code.
+     */
+    struct aws_custom_key_op_handler key_handler;
```

Comment:
trivial: typically when making "subclasses" in C we'd put this as the first member and name it "base"